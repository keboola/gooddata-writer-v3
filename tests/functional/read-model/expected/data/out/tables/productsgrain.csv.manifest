{
    "destination": "in.c-gd-model.productsgrain"
}