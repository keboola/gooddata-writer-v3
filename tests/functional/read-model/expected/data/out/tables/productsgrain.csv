a_id,info,a_name,f_price,datasetoutcmaincategories,productdate